python/lsst/validate/drp/plot.py

```diff
@@ -491,8 +491,11 @@ def plotAMx(job, amx, afx, filterName, amxSpecName='design', outputPrefix=""):
 
     Parameters
     ----------
-    amx : `AMxMeasurement`
-    afx : `AFxMeasurement`
+    job : `lsst.verify.Job`
+        `Job` providing access to metrics, specs and measurements
```

Comment:
I think for this hyperlink to work, and still just read as "Job", you'll want to do:

```rst
`~lsst.verify.Job`
```